### PR TITLE
Ah add doc how to contribute

### PR DESCRIPTION
**Pull request by andistorm**

### CODE REVIEW

**Comment by caller** [comment]
I would not entirely delete the contributing.md. Seems to be a standard file one would look for. Just keep it there but fill it with "please look on this other place: LINK" like content

**Comment by caller** [comment]
please add some more explicit content about how to report securty related issues. I think there is some explicit place (or .md) file where the process is described. we shoul point that out a bit more often?

**Comment by caller** [comment]
DCO check is failing here, so by LFE policy we cant / shouldnt merge

File: docs/how_to_contribute/how_to_contribute_overview.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. _how_to_contribute_overview:
+
+How to contribute
+#################
+*Current status of Q1 2022*
```
Comment (SebaLukas) [inline docs/how_to_contribute/how_to_contribute_overview.rst]: Perhaps Q3 2022?

File: docs/how_to_contribute/how_to_work_on_issues.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Issue states
+************
+
+* Unconfirmed
+    New created public issues are unconfirmed. The issue can be moved by team members to
+    `Untriangled`_ by validating, reproducing or otherwise that this is a bug or a feature request.
+
+* Untriaged
+    Conirmed issues that haven't been reviewed for priority or assignment. This is the default
```
Comment (SebaLukas) [inline docs/how_to_contribute/how_to_work_on_issues.rst]: Confirmed instead of Conirmed

File: docs/how_to_contribute/how_to_work_on_issues.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+How To: Work on issues
+#######################
+
+Issue states
+************
+
+* Unconfirmed
+    New created public issues are unconfirmed. The issue can be moved by team members to
+    `Untriangled`_ by validating, reproducing or otherwise that this is a bug or a feature request.
```
Comment (LAD101work) [inline docs/how_to_contribute/how_to_work_on_issues.rst:11]: This seems to rather be `Untriaged`, right?

File: docs/how_to_contribute/how_to_work_on_issues.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    Assigned issues that are currently being worked on.
+    The issue can be moved to `In Approving / Fixed`_ after it has been finished and needs
+    to be approved by a team member.
+
+* In Approving / Fixed
+    Issues that are being reviewed by the team.
+    The issue can be moved to `Closed`_ after a team member has approved it and merged all linked pull requests.
+
+* Closed
+    Issues that are done or closed by other reasons.
```
Comment (LAD101work) [inline docs/how_to_contribute/how_to_work_on_issues.rst]:
```suggestion
    Issues that are done or closed for other reasons.
```

File: docs/how_to_contribute/how_to_work_on_issues.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+#. :ref:`Create a WIP pull request <how_to_pull_request>` for each branch related to the issue and link it to the issue
+
+#. After you have finished working on the issue, move it to `In Approving / Fixed`_
+
+    Make sure you are still assigned to the issue. Assign a team member as reviewer and wait for them 
+    to review your code. If no code changes are requested and the issue is approved by a team member, the 
+    pull request will be merged and the issue will be moved to `Closed`_.
+
+#. After a team meber has approved the issue, it should be moved to `Closed`_ automatically by merging all linked pull requests.
```
Comment (LAD101work) [inline docs/how_to_contribute/how_to_work_on_issues.rst]:
```suggestion
#. After a team member has approved the issue, it should be moved to `Closed`_ automatically by merging all linked pull requests.
```

File: docs/how_to_contribute/how_to_work_on_issues.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
+.. _how_to_work_on_issues:
+
+How To: Work on issues
+#######################
+
+Issue states
+************
+
+* Unconfirmed
+    New created public issues are unconfirmed. The issue can be moved by team members to
```
Comment (LAD101work) [inline docs/how_to_contribute/how_to_work_on_issues.rst]:
```suggestion
    Newly created public issues are unconfirmed. The issue can be moved by team members to
```

**Pull request closed without merging**